src/components/inspectors/PreviewPanel.vue

```diff
@@ -117,6 +109,12 @@ export default {
     loading() {
       this.showSpinner = false;
     },
+    openAsset() {
+      const nodeConfig = this.previewConfigs.find(config => {
+        return config.matcher(this.data);
+      });
```

Comment:
I would recommend taking out this piece of code into his own function to avoid a repetitive function